go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/algorand/msgp v1.1.47
 	github.com/algorand/oapi-codegen v1.3.5-algorand5
 	github.com/algorand/websocket v1.4.2
+	github.com/algorand/xorfilter v0.0.0-20210423194602-fadd755a812a
```

Comment:
we need to create a release for the xor filter and use that release version here.